server/lib/validation/languages/th.js

```diff
@@ -108,8 +105,26 @@ const INVALIDATIONS = [{
   regex: /[\u0E30][\u0E32\u0E33\u0E45]/,
   error: 'Sentence should not contain Sara Aa, Sara Am or Lakkhangyao after Sara A',
 }, {
-  regex: /[\u0E01-\u0E4Ea-zA-Z.,\-"'“”‘’\u0060?!:;]{55}/,
-  error: 'Sentence should not contain more than 54 characters running without whitespace',
+  // 71 or more consonants/vowels running without a space is difficult to read
+  regex: /[\u200b\u200c\u2063\u0E01-\u0E4E]{71}/,
+  error: 'Sentence should not contain more than 70 consonants and vowels running without a space',
+}, {
+  // 81 or more characters running wihtout a space is difficult to read
+  regex: /[\u200b\u200c\u2063\u0E01-\u0E4E.,\-"'“”‘’\u0060?!:;]{81}/,
+  error: 'Sentence should not contain more than 80 characters running without a space',
+}, {
+  // 31 or more repeating consonants running without a space likely difficult to read.
+  regex: /[\u200b\u200c\u2063ก-ฮ]{31}/,
+  error: 'Sentence should not contain more than 30 of consonants running wihtout a space',
```

Comment:
minor: small typo in "whtout" -> "without"

```suggestion
  error: 'Sentence should not contain more than 30 consonants running without a space',
```